LAION-AI/Open-Assistant
# Web App Manifest - Save as desktop or mobile app

From issue https://github.com/LAION-AI/Open-Assistant/issues/3548
✅ Branched from latest
✅ Pre-commit changes added

This commit adds a manifest file that allows users to save the website on desktop or mobile, see [this documentation](https://developer.mozilla.org/en-US/docs/Web/Manifest).
It makes an icon that can be added to dock, mobile homescreen, etc.

Changes in this PR:
- Add a smaller logo file for certain mobile platforms
- Add manifest.json that contains logos, titles, descriptions, etc.
- Add a link to manifest.json in _document.tsx

<img width="700" alt="Screenshot 2023-08-03 at 8 23 43 PM" src="https://github.com/LAION-AI/Open-Assistant/assets/80140457/f86946cb-62d7-42a6-a9ed-4a493b91d1da">
<img width = "300" alt ="mobile icon" src = "https://github.com/LAION-AI/Open-Assistant/assets/80140457/8f81a8ba-3c57-4fb1-98a0-b20a37a188bd"/>
